deployments/charts/kuma/README.md

```diff
@@ -61,6 +61,8 @@ A Helm chart for the Kuma Control Plane
 | controlPlane.extraSecrets | list | `[]` | Additional secrets to mount into the control plane |
 | controlPlane.webhooks.validator.additionalRules | string | `""` | Additional rules to apply on Kuma validator webhook. Useful when building custom policy on top of Kuma. |
 | controlPlane.webhooks.ownerReference.additionalRules | string | `""` | Additional rules to apply on Kuma owner reference webhook. Useful when building custom policy on top of Kuma. |
+| controlPlane.podSecurityContext | string | `nil` | Security context at the pod level for control plane. |
+| controlPlane.containerSecurityContext | string | `nil` | Security context at the container level for control plane. |
```

Comment:
Should these be "object" as a type we add something similar with `resources` and we just changed it from string to object. Sounds to me that it should be the same here. See: https://github.com/kumahq/kuma/pull/4049/files